src/render/pic.rs

```diff
@@ -0,0 +1,72 @@
+//! PIC rendering support.
+//!
+//! # Example
+//!
+//! ```
+//! use qrcode::QrCode;
+//! use qrcode::render::pic;
+//!
+//! let code = QrCode::new(b"Hello").unwrap();
+//! let pic = code.render::<pic::Color>().build();
+//! println!("{}", pic);
+
+#![cfg(feature = "pic")]
+
+use std::fmt::Write;
+use std::marker::PhantomData;
+
+use crate::render::{Canvas as RenderCanvas, Pixel};
+use crate::types::Color as ModuleColor;
+
+/// A PIC color.
+#[derive(Copy, Clone, Default, PartialEq, Eq, PartialOrd, Ord, Hash)]
+pub struct Color<'a>(pub &'a str);
```

Comment:
is the `&str` member still needed?

Reply:
I'm afraid answering that question exceeds my knowledge of Rust 🙁 I'd say "No, it's no longer needed", but trying to remove it results in compiler errors I'm unable to resolve. Would you be willing to help out here, @kennytm?

Reply:
you could make it a unit struct

```rust
pub struct Color;
```

and remove all mentions of `'a` in this file.

Reply:
Thanks for the helpful suggestion, @kennytm, I've made the appropriate changes to this PR.